SharkCage/packages/Newtonsoft.Json.11.0.2/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
This file should not be tracked, we might have to change the order of our .gitignore entries to have the allowed file types at the top